AmountInput/README.md

```diff
@@ -0,0 +1,47 @@
+
+# AmountInput Interface
+
+**Status**: Proposed
+
+| Name        | ID                                                                |
+| :---------- | :---------------------------------------------------------------- |
+| AmountInput | a1d347099e29c1624c8890619daf207bde18e92df5220a54bcc6d858309ece84  |
+
+
+## Description
+
+Interface for amount input.
+
+## Functions
+
+`get` - prints prompt message to the user and returns number of nanotons entered by user.
+
+arguments:
+
+	answerId: uint32
+    prompt: bytes
+    step: uint256
+	min: uint256
```

Comment:
What about negative values? Aren't they possible?

Reply:
Do we need a negative amount of tokens?

Reply:
Don't know, but I can imagine an example of such a case. Maybe it's redundant indeed.

Reply:
I think that negative number is incorrect param.